### PR TITLE
updated the documentation for the vertical divider component

### LINKED ISSUE #10844: [docs]: Update the documentation for the Vertical Divider Component
### Summary

**The entire URL of the documentation with the issue**

https://docs.tooljet.com/docs/widgets/vertical-divider

**Formatting updates:**
- Update the formatting based on the [latest](https://docs.tooljet.com/docs/contributing-guide/documentation-guidelines/style-guide) guidelines.
- For each h2 (created using ##), remove 24px padding-bottom. The `div` that wraps up the h2 headers and the section below the header should only have a "padding-top" of 24px and no "padding-bottom". 

**Content updates:**
- Add a more descriptive introduction
- Replace the word "widget" with "component" wherever applicable. 
- Layout Table
  - Replace the header "Layout" with "Devices"
  - Update the Description and Expected Value columns [reference](https://docs.tooljet.com/docs/widgets/text-input#devices)
 - Styles Table
   - Update the styles table and its content with three headers - Property, Description, Configuration Options [reference](https://docs.tooljet.com/docs/widgets/text-input#label) 
   - Include box-shadow under the Styles table 

**The changes are to be implemented in two files -** 
 Next - ToolJet/docs/docs/widgets/vertical-divider.md
 Version 2.50.0 (LTS) - ToolJet/docs/versioned_docs/version-2.50.0-LTS/widgets/vertical-divider.md

 _**The deadline for this task is 3 days from the day of the assignment. Please feel free to reach out to us in our [Slack community](https://tooljet.com/slack) if you have any questions.**_
   
**If the issue is confirmed, would you be willing to submit a pull request?**

No

### Code of Conduct

- [X] I agree to follow the ToolJet Code of Conduct

### PR DESCRIPTION
**Pull request by PARVNEMA**
i have updated the documentation for the vertical divide component .
fixes #10844

### CODE REVIEW

**Comment by RudraPratik30** [comment]
Hey @PARVNEMA!!

Can you please correct the syntax on line no. 17?
There should be two closing curly braces, in both the version.
Please do it, so that we can deploy the PR on render and test it out.

**Comment by PARVNEMA** [comment]
Ok i am doing it

**Comment by PARVNEMA** [comment]
@RudraPratik30  done it sir

**Comment by RudraPratik30** [comment]
Hey, you just made the changes in only one version, can you edit the next version as well?

**Comment by PARVNEMA** [comment]
@RudraPratik30  sir i have made changes in both the versions

**Comment by PARVNEMA** [comment]
@RudraPratik30  done it sir and sorry this is my first open source contribution and i am learning

**Comment by RudraPratik30** [comment]
No worries @PARVNEMA, It's alright!

File: docs/docs/widgets/vertical-divider.md [hunk 1]
```diff
@@ -4,58 +4,59 @@ title: Vertical Divider
...
-| <div style={{ width:"100px"}}> Properties </div> | <div style={{ width:"100px"}}> Description </div> |
-| ----------- | ----------- |
-| Divider Color |  It is used to set the color of the divider. Use hex code to set the background color. |
-| Visibility |  This property is used to set the visibility of the divider. The property accepts Boolean value. |
+## Styles
 
+| <div style={{ width:"120px"}}> Property </div> | <div style={{ width:"150px"}}> Description </div> | <div style={{ width:"180px"}}> Configuration Options </div> |
+| ----------- | ----------- | ----------- |
+| Divider Color | Sets the color of the divider. Use hex code or color names. | Accepts any valid color value like hex (`#000000`), RGB, or predefined color names. |
+| Visibility | Controls whether the divider is visible. Accepts Boolean values. | `true` (visible), `false` (hidden) |
```
Comment (RudraPratik30) [inline docs/docs/widgets/vertical-divider.md:55]: Row: Visibility, Column: Configuration Options
Please update the content, include how we can use the fx function there, check out the [reference](https://docs.tooljet.com/docs/widgets/text-input/#label).

File: docs/docs/widgets/vertical-divider.md [hunk 2]
```diff
@@ -4,58 +4,59 @@ title: Vertical Divider
...
+| Box Shadow | Adds shadow around the divider to create depth. | Accepts `box-shadow` values like `none`, `2px 4px 6px rgba(0, 0, 0, 0.1)` |
```
Comment (RudraPratik30) [inline docs/docs/widgets/vertical-divider.md:56]: Row: Box Shadow, Column: Configuration Options
box-shadow should be a normal text and should not be enclosed in single ticks

File: docs/docs/widgets/vertical-divider.md [hunk 3]
```diff
@@ -4,58 +4,59 @@ title: Vertical Divider
...
+<div style={{paddingTop:'24px'}}>
 
 ## General
 ### Tooltip
 
-A Tooltip is often used to specify extra information about something when the user hovers the mouse pointer over the widget.
+A Tooltip is often used to specify extra information about something when the user hovers the mouse pointer over the component.
 
-Under the <b>General</b> accordion, you can set the value in the string format. Now hovering over the widget will display the string as the tooltip.
+Under the <b>General</b> accordion, you can set the value in the string format. Now hovering over the component will display the string as the tooltip.
```
Comment (RudraPratik30) [inline docs/docs/widgets/vertical-divider.md:32]: Use * to mark something bold instead of using HTML tags

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]
Hey, Please make the MENTIONED changes in both the versions.

**Comment by PARVNEMA** [comment]
Ok will apply changes and send

**Comment by PARVNEMA** [comment]
@RudraPratik30  done changes

**Review by RudraPratik30 — APPROVED** [review]
Hey @parvnema approving the changes from my end and forwarding it to the team for final review.
Thanks for contributing! Your PR will be merged by the end of this week.

**Comment by PARVNEMA** [comment]
Thanks @RudraPratik30  for this opportunity and if any more issue please tell me

**Review by RudraPratik30 — APPROVED** [review]

**Review by karan-rathod-316 — APPROVED** [review]
Looks good to me.

**Pull request merged**